scripts/dev/misc_checks.py

```diff
@@ -226,6 +231,10 @@ def check_spelling(args: argparse.Namespace) -> Optional[bool]:
             re.compile(r'IOError'),
             "use OSError",
         ),
+        (
+            re.compile(r'qtbot\.(?!{})[a-z]+[A-Z].*'.format(excluding)),
+            "Snake-case available",
```

Comment:
```suggestion
            "use snake-case instead",
```

to be consistent with the other messages.